### PR TITLE
Update Get-ADGroup.md

### PR DESCRIPTION
**Pull request by msbemba**
Canonical name is not accepted value for identity parameter in get-adgroup

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2680

File: docset/winserver2012-ps/activedirectory/Get-ADGroup.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Get-ADGroup [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <S
 The Get-ADGroup cmdlet gets a group or performs a search to retrieve multiple groups from an Active Directory.
 
 The Identity parameter specifies the Active Directory group to get.
-You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name, or canonical name.
+You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADGroup.md:39]:
```suggestion
You can identify a group by its distinguished name (DN), GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2012r2-ps/activedirectory/Get-ADGroup.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Get-ADGroup [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <S
 The **Get-ADGroup** cmdlet gets a group or performs a search to retrieve multiple groups from an Active Directory.
 
 The **Identity** parameter specifies the Active Directory group to get.
-You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name, or canonical name.
+You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/activedirectory/Get-ADGroup.md:41]:
```suggestion
You can identify a group by its distinguished name (DN), GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2016-ps/activedirectory/Get-ADGroup.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Get-ADGroup [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <S
 The **Get-ADGroup** cmdlet gets a group or performs a search to retrieve multiple groups from an Active Directory.
 
 The *Identity* parameter specifies the Active Directory group to get.
-You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name, or canonical name.
+You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/activedirectory/Get-ADGroup.md:42]:
```suggestion
You can identify a group by its distinguished name (DN), GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2019-ps/activedirectory/Get-ADGroup.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ Get-ADGroup [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <S
 The **Get-ADGroup** cmdlet gets a group or performs a search to retrieve multiple groups from an Active Directory.
 
 The *Identity* parameter specifies the Active Directory group to get.
-You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name, or canonical name.
+You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADGroup.md:42]:
```suggestion
You can identify a group by its distinguished name (DN), GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2022-ps/activedirectory/Get-ADGroup.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ Get-ADGroup [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <S
 The **Get-ADGroup** cmdlet gets a group or performs a search to retrieve multiple groups from an Active Directory.
 
 The *Identity* parameter specifies the Active Directory group to get.
-You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name, or canonical name.
+You can identify a group by its distinguished name (DN), GUID, security identifier (SID), Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/Get-ADGroup.md:42]:
```suggestion
You can identify a group by its distinguished name (DN), GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**